.github/dependabot.yml

```diff
@@ -0,0 +1,26 @@
+# See the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "daily"
+
+  - package-ecosystem: "gomod"
+    directory: "/tools"
+    schedule:
+      interval: "daily"
+
+  - package-ecosystem: "github-actions"
+    # Workflow files stored in the
+    # default location of `.github/workflows`
+    directory: "/"
+    schedule:
+      interval: "daily"
+
+  - package-ecosystem: "cargo"
+    directory: "/"
```

Comment:
does it work even if the cargo part is in a subdirectory?

Reply:
Technically, the cargo part is on the root as well... If we configure the workspace members correctly (which we did, otherwise everything would be breaking), it works perfectly. We have the same setup in `aad-auth`.

Reply:
ack then?